system_strings/sysmes_text.en

```diff
@@ -3173,19 +3173,24 @@ Possessed prison of shadow
 }
 [sha:3339a3ffe3d59df0b5b9a8ba57eb5a1369f3a82a]{
 -- 15日目―――
-15th day――
+// @flowchart_title
+15th day
 }
 [sha:6c822ae19291fffdbdaa9c3f42beaefe300ff9e1]{
 -- 告白
+// @flowchart_title
 Confession
 }
 [sha:ec49e960c0ff39e25a26c3c0032dc6f3ecb26ddc]{
 -- 旅の終わり
-End of the journey
+// @flowchart_title
+Journey's End
 }
 [sha:e6c3fb2fcff0597063d68558af3b70320fe21476]{
 -- 月の荒野。
-Moon Wilderness.
+// Same phrase used in 32352.
+// @flowchart_title
+Barren Surface of the Moon
 }
 [sha:dde5aecaaeefdac2ea934226b1985f1bb0f8875b]{
 -- 吸血姫のキス
```

Comment:
Reason this one's like this is because of how it pairs with the subtitle - 
> I kiss the vamppiress
> So I may never forget the moon's light.

Definitely prefer suggestion as a title if we can find some way to change the second sentence / are okay with it just not being that connected.

Reply:
That effect isn't there in the Japanese and I don't think 'titles' should work like that anyway